src/modules/schedule/components/calendar/calendar.tsx

```diff
@@ -64,11 +63,50 @@ export const Calendar: FC<CalendarProps> = () => {
           </div>
         )}
         allDaySlot={false}
-        // events={events}
         height={'75vh'}
-        buttonText={{ today: 'Today' }}
         nowIndicator
         slotMinTime={'07:00'}
+        events={events.map((event) => ({
+          id: event.id,
+          title: event.serviceProfessional.title,
+          start: set(event.date, {
+            hours: getHours(event.startTime),
+            minutes: getMinutes(event.startTime),
+            seconds: 0,
+            milliseconds: 0,
+          }),
+
+          end: set(event.date, {
+            hours: getHours(event.endTime),
+            minutes: getMinutes(event.endTime),
+            seconds: 0,
+            milliseconds: 0,
+          }),
+
+          className: styles.event,
+        }))}
```

Comment:
In this particular case I think it would be nice to move this map to the separate variable nad wrap it in useMemo.